packages/controllers/src/controllers/ConnectorController.ts

```diff
@@ -224,9 +224,12 @@ export const ConnectorController = {
   getConnectorById(id: string) {
     return state.allConnectors.find(c => c.id === id)
   },
-
   getConnector(id: string, rdns?: string | null) {
-    return state.allConnectors.find(c => c.explorerId === id || c.info?.rdns === rdns)
+    const connectorsByNamespace = state.allConnectors.filter(
```

Comment:
```suggestion
    const activeNamespaceConnectors = state.allConnectors.filter(
```
maybe which indicates what does the filter do exactly - but not super imp